libwallet/src/slatepack/packer.rs

```diff
@@ -48,11 +48,6 @@ impl<'a> Slatepacker<'a> {
 
 	/// return slatepack
 	pub fn deser_slatepack(&self, data: Vec<u8>, decrypt: bool) -> Result<Slatepack, Error> {
```

Comment:
Is this called from places other than `get_slatepack_file_contents` ?
We may want to keep the size validation here to handle other non-file based cases.